frontend/WikiContrib-Frontend/src/components/activity.js

```diff
@@ -72,39 +98,41 @@ class Activity extends React.Component {
           <React.Fragment>
             {this.state.data.length !== 0 ? (
               <React.Fragment>
-                <h4>User Activity on {this.props.date}</h4>
+                <h4>{this.props.username}'s activity on {this.props.date}</h4>
                 {this.state.data.map((obj, index) => (
                   <Card className="commits_load" key={index}>
                     <Card.Content>
-                      {obj.platform === 'Phabricator' ? (
-                        <a
-                          href={
-                            'https://phabricator.wikimedia.org/' + obj.redirect
-                          }
-                          target="_blank"
-                          rel="noopener noreferrer"
-                        >
-                          <h3>{obj.heading}</h3>
-                        </a>
-                      ) : (
-                        <a
-                          href={
-                            'https://gerrit.wikimedia.org/r/#/q/' + obj.redirect
-                          }
-                          target="_blank"
-                          rel="noopener noreferrer"
-                        >
-                          <h3>{obj.heading}</h3>
-                        </a>
-                      )}
-                      <div>
-                        <span style={{ display: 'inline', float: 'left' }}>
-                          <b>PLATFORM:</b> {obj.platform}
-                        </span>
-                        <span style={{ display: 'inline', float: 'right' }}>
```

Comment:
I kind of like how we were displaying the status with the 'Status' keyboard earlier and the status font was light in weight too.